index.html

```diff
@@ -173,126 +173,364 @@ <h2>
     </section>
     <section>
       <h2>
-        <code>pointerlockchange</code> and <code>pointerlockerror</code> Events
+        Pointer Lock and Interfaces
       </h2>
-      <p>
-        Two events are used to communicate pointer lock state change or an
-        error in changing state. They are named <dfn>pointerlockchange</dfn>
-        and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
-        for any reason a <a>pointerlockchange</a> event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>Pointer Lock State</dfn> Definition
+        </h3>
+          <p><a>Pointer Lock State</a> is the state where a single DOM element, which
+            we will call the <dfn>pointer-lock target</dfn>, receives all mouse events and the cursor
+            is hidden.</p>
+          <p>
+            Once in the <a>Pointer Lock State</a> the <a>user agent</a> has a
+            <a>pointer-lock target</a> and a set of <a>PointerLockOptions</a>. The
+            <a>pointer-lock target</a> must receive all relevant user generated
+            {{MouseEvent}} events (for example: `mousemove`, `mousedown`, `mouseup`, 
+            `click`, and `wheel`) [[ui-events]]. No other elements may receive these
+            events while in <a>Pointer Lock State</a>. Events that require the concept of
+            a mouse cursor must not be dispatched (for example: `mouseover`, `mouseout`,
+            `drag`, and `drop`).
```

Comment:
Would it be possible to list these events normatively, instead of only giving some examples?

I think the current text is OK, because the lack of precision here is just a symptom of the larger lack of precision in how mouse events are specified in general. Some progress is being made there, in https://github.com/w3c/uievents/issues/200, and if that lands then eventually this paragraph can move into the UI events spec as an actual change to the processing model.

But in the meantime, it would help achieve interoperability if you were precise about the list of events.

For example, it's not clear to me whether pointer events are impacted by this change.

Reply:
@alvinjiooo Please add a link here to this issue I just filed: https://github.com/w3c/pointerlock/issues/97

Reply:
Done